core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -114,6 +114,108 @@ oppia.factory('UrlInterpolationService', ['alertsService',
         match = filledUrl.match(INTERPOLATION_VARIABLE_REGEX);
       }
       return filledUrl;
+    },
+
+    /**
+     * Given an resource path, returns the relative url path to that resource
+     * prefixing the appropriate cache_slug to it.
+     */
+    getStaticResourceUrl: function(resourcePath) {
+      if (!resourcePath) {
+        alertsService.fatalWarning(
+          'Empty resource path passed in getStaticResourceUrl.');
+        return null;
+      }
+
+      var RESOURCE_PATH_STARTS_WITH_FORWARD_SLASH = /^\//;
+      // Ensure that resourcePath starts with a forward slash.
+      if (!resourcePath.match(RESOURCE_PATH_STARTS_WITH_FORWARD_SLASH)) {
+        alertsService.fatalWarning(
+          'Resource path passed into getStaticResourceUrl must start with' +
+          '\'\/\': \'' + new String(resourcePath) + '\'.');
+        return null;
+      }
+
+      return GLOBALS.ASSET_DIR_PREFIX + resourcePath;
+    },
+
+    /**
+     * Given a domain url and resource path, returns the complete url path
+     * to that resource prefixing the appropriate cache_slug to it.
+     */
+    getCompleteStaticResourceUrl: function(domainUrl, resourcePath) {
```

Comment:
If not used, let's remove it. Otherwise it is just dead code.

On Saturday, July 16, 2016, Vishal Gupta <notifications@github.com> wrote:

> In core/templates/dev/head/domain/utilities/UrlInterpolationService.js
> <https://github.com/oppia/oppia/pull/2208#discussion_r71070124>:
>
> > +      // Ensure that resourcePath starts with a forward slash.
> > +      if (!resourcePath.match(RESOURCE_PATH_STARTS_WITH_FORWARD_SLASH)) {
> > +        alertsService.fatalWarning(
> > +          'Resource path passed into getStaticResourceUrl must start with' +
> > +          '\'\/\': \'' + new String(resourcePath) + '\'.');
> > +        return null;
> > +      }
> > +
> > +      return GLOBALS.ASSET_DIR_PREFIX + resourcePath;
> > +    },
> > +
> > +    /**
> > +     * Given a domain url and resource path, returns the complete url path
> > +     * to that resource prefixing the appropriate cache_slug to it.
> > +     */
> > +    getCompleteStaticResourceUrl: function(domainUrl, resourcePath) {
>
> I thought of using this for the meta tags, but then couldn't use it.
> Should I leave it or remove it?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/2208/files/aa4be4efc52f62354013d697197d9835201afd39#r71070124>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKFeyr1u__5hgf3TwDEOyRsfGINeOacPks5qWSawgaJpZM4JDv96>
> .
>